scripts/install/linux_test_dependencies.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ $EUID -ne 0 ]]; then
+       echo "This script must be run as root"
+       exit 1
+fi
+
+UBUNTU_VERSION=$(lsb_release -rs)
+if [[ $UBUNTU_VERSION == "16.04" ]]; then
+       export ROS_DISTRO=kinetic
+elif [[ $UBUNTU_VERSION == "18.04" ]]; then
+       export ROS_DISTRO=melodic
+elif [[ $UBUNTU_VERSION == "20.04" ]]; then
+       export ROS_DISTRO=noetic
+else
+       echo "Unsupported Linux version."
+fi
```

Comment:
Note that it is wrong to say that ROS is only a test dependency as it is actually required for the compilation of the ros controller on linux.

Reply:
Yes, but it is still part of the optional compilation dependencies: the optional compilation dependencies includes the test dependencies.

Reply:
It indeed looks weird :) I guess the intuitive solution (to read) would be something like:
```
|-- linux_optional_compilation_dependencies.sh
    |-- linux_ros_dependency.sh
    |--  linux_test_dependencies.sh
         |-- linux_ros_dependency.sh (because of check_urdf)
         |-- check_urdf
```
`linux_ros_dependency.sh` is executed twice, but it is not a big deal.

However, this is overkill for us, so I guess we can keep it as is.

Reply:
Note: that `linux_test_dependecies.sh` is not executed twice in this PR as I implemented a `--norecurse` option.

Reply:
Or simply, create `linux_ros_dependency.sh` and use it in the workflow directly + `check_urdf` installation.

Reply:
> Or simply, create `linux_ros_dependency.sh` and use it in the workflow directly + `check_urdf` installation.

I don't like this solution, as dependencies should be organized in a hierarchical way so that when you need some dependency, you shouldn't bother about managing yourself the sub-dependencies of it.